lib/developer_portal/lib/liquid/filters/rails_helpers.rb

```diff
@@ -11,7 +11,7 @@ module RailsHelpers
 
       THREESCALE_STYLESHEETS = %w[legacy/stats plans_widget.css active-docs/application.css stats.css].freeze
       THREESCALE_JAVASCRIPTS = %w[buyer/1/analytics plans_widget.js plans_widget_v2.js active-docs/application.js stats.js].freeze
-      THREESCALE_WEBPACK_PACKS = %w[stats.js active_docs.js].freeze
+      THREESCALE_WEBPACK_PACKS = %w[stats.js active_docs.js loadStripe.js].freeze
```

Comment:
Could we do `load_stripe.js` instead?

Reply:
hmmm, there is no rule, in `javascript/packs` we are using `camelCase`, `snake_case` and even `kebab-case`, changing to `load_stripe` will not fix that in the `packs` directory, but at least will be more consistent in this file, changing it.